### PR TITLE
resetting consumer group offset has been updated 

### PR DESCRIPTION
**Pull request by fmcdonal**
resetting consumer group offset has been updated to appear as it does in the UI and replaced custom offset with absolute

### CODE REVIEW

**Comment by fmcdonal** [comment]
@mmurphy please review

File: docs/kafka/consumer-configuration-kafka/README.adoc [hunk 1]
```diff
@@ -291,11 +291,12 @@ Use the {product-kafka} web console to select consumer groups and reset partitio
 A reset changes the offset position from which consumers read from the message log of a topic partition.
 To reset an offset position, the consumer group must have _NO MEMBERS_ connected to a topic.
 
-Choose one of the following options for the reset:
+Choose one of the following options for a *New offset*:
```
Comment (jbyrne-redhat) [inline docs/kafka/consumer-configuration-kafka/README.adoc:294]: Suggest: `Choose one of the following options for *New offset*:` (no article)

File: docs/kafka/consumer-configuration-kafka/README.adoc [hunk 2]
```diff
@@ -291,11 +291,12 @@ Use the {product-kafka} web console to select consumer groups and reset partitio
...
 
-* *Earliest* resets to the earliest offset at the start of the message log.
-* *Latest* resets to the latest offset at the end of the message log.
-* *Custom offset* resets to a specific offset in the message log.
+
+* *absolute* resets to a specific offset in the message log.
```
Comment (jbyrne-redhat) [inline docs/kafka/consumer-configuration-kafka/README.adoc:297]: Are these names definitely in lowercase in the UI?

Comment (fmcdonal) [reply docs/kafka/consumer-configuration-kafka/README.adoc:297]: yes I generated the steps, that's why I changed to lowercase

**Review by jbyrne-redhat — APPROVED** [review]

**Pull request merged**